userspace/libscap/linux/scap_cgroup.c

```diff
@@ -496,6 +496,11 @@ static int32_t get_cgroup_subsystems_v2(struct scap_cgroup_interface* cgi, struc
 
 	char line[SCAP_MAX_PATH_SIZE];
 	snprintf(line, sizeof(line), "%s/cgroup.controllers", cgroup_mount);
+	if (access(line, F_OK) == -1) {
+        // If the file does not exist, return success. Skip
+        return SCAP_SUCCESS;
+    }
```

Comment:
Super nitpicky, but a bit of indentation would be nice.
```suggestion
	if (access(line, F_OK) == -1) {
		// If the file does not exist, return success. Skip
		return SCAP_SUCCESS;
	}
```

Reply:
Thank you for the adjustment. The local editor is working properly, but there is an indentation issue when submitting to GitHub